xml/System.Threading/CancellationTokenSource.xml

```diff
@@ -383,8 +383,10 @@
 ## Remarks  
  The associated <xref:System.Threading.CancellationToken> will be notified of the cancellation and will transition to a state where <xref:System.Threading.CancellationToken.IsCancellationRequested%2A> returns true.  
   
- Any callbacks or cancelable operations registered with the <xref:System.Threading.CancellationToken> will be executed.
-  
+ Any callbacks or cancelable operations registered with the <xref:System.Threading.CancellationToken> will be executed, if they have not already been executed by a previous call to <xref:System.Threading.CancellationTokenSource.Cancel>. Subsequent calls to <xref:System.Threading.CancellationTokenSource.Cancel> will not execute the same callback again unless re-registered. (We recommend avoiding multiple calls to <xref:System.Threading.CancellationTokenSource.Cancel>, because the intent of such code is often unclear.)
```

Comment:
Didn't really know the corporate-speak phrasing for this. When writing for a human, one would say "Try not to call `Cancel` multiple times, because it breaks the mental model that the `IsCancellationRequested` state is terminal, and is therefore confusing to read".